adapters/handlers/rest/configure_api.go

```diff
@@ -588,8 +595,8 @@ func configureAPI(api *operations.WeaviateAPI) http.Handler {
 	appState := MakeAppState(ctx, connectorOptionGroup)
 
 	appState.Logger.WithFields(logrus.Fields{
-		"server_version":   config.ServerVersion,
-		"docker_image_tag": config.ImageTag,
+		"server_version": config.ServerVersion,
+		"version":        build.Version,
```

Comment:
I replaced `docker_image_tag` -> just simple `version` because

1. It works for every case, even we are running it as container or normal local process. Exposing "docker_*` seems irrelevant.
2. Plus, [what we set for `docker_image_tag` even before it's indeed the version.](https://github.com/weaviate/weaviate/blob/main/tools/generate-release-artifacts.sh#L51)